src/repositories/likes.repository.ts

```diff
@@ -0,0 +1,32 @@
+import { LikeModel } from "../models/Like.model";
+
+export const getOne = async (
+  data: { _id: string } | { userId: string; tweetId: string }
+) => {
+  return LikeModel.findOne(data);
+};
+export const deleteOne = async (
+  data:
+    | {
+        _id: string;
+      }
+    | {
+        userId: string;
+        tweetId: string;
+      }
+) => {
+  return LikeModel.deleteOne(data);
+};
+export const createOne = async (tweetId: string, userId: string) => {
+  const likeModel = new LikeModel({
+    userId,
+    tweetId,
+  });
+  return likeModel.save();
+};
+export const updateOne = async (
```

Comment:
```suggestion
export const updateOne =  (
```
no await in that function